src/components/molecules/ChatsList/ChatsList.tsx

```diff
@@ -193,7 +194,7 @@ const ChatsList: React.FunctionComponent = () => {
                       }`}
                     >
                       {formatDistanceToNow(
-                        new Date(lastMessageExchanged.ts_utc.toDate())
+                        lastMessageExchanged.ts_utc.toDate()
```

Comment:
[nit] since `formatDistanceToNow` accepts a `number`, I believe it should be sufficient to do

```suggestion
                        formatDistanceToNow(lastMessageExchanged.ts_utc)
```